packages/react-components/react-input/src/components/Input/useInputStyles.styles.ts

```diff
@@ -288,26 +344,30 @@ export const useInputStyles_unstable = (state: InputState): InputState => {
   state.input.className = mergeClasses(
     inputClassNames.input,
     useInputClassName(),
-    size === 'large' && inputStyles.large,
+    (size === 'large' && inputStyles.large) || (size === 'small' && inputStyles.small),
```

Comment:
You can add an empty `medium` key to inputStyles (`medium: {}`) and then just index this on `size`

```suggestion
    inputStyles[size],
```